### ISSUE TITLE
Inheriting from TypedDict/NamedTuple is flagged as invalid

### ISSUE THREAD

**Issue by gaborbernat** [PyCQA/pylint#3884]
See https://bugs.python.org/issue41973, the type of TypedDict/NamedTuple is now function, which apparently is a valid object to inherit from.

**Comment by hippo91** [comment]
@gaborbernat is your problem related to `pylint`? If so please give us a snippet that illustrates it and follow the template for bug submission.

**Comment by gaborbernat** [comment]
```python
from typing import TypedDict, NamedTuple

class CustomDict(TypedDict):
    alpha: int

class CustomTuple(NamedTuple):
    beta: int
```

```
pylint 'magic.py'
************* Module magic
magic.py:3:0: E0239: Inheriting 'TypedDict', which is not a class. (inherit-non-class)
methods)
magic.py:6:0: E0239: Inheriting 'NamedTuple', which is not a class. (inherit-non-class)
```

**Comment by hippo91** [comment]
@gaborbernat this issue it the same as #3876. I close this issue.
